test/views/vatDetails/VatDetailsViewSpec.scala

```diff
@@ -258,6 +259,47 @@ class VatDetailsViewSpec extends ViewBaseSpec {
         updateVatDetails.select("p").text() shouldBe "Change your business, contact or VAT details."
       }
     }
+
+    "not have the mtd signup section" in {
+      elementExtinct(Selectors.mtdSignupSection)
+    }
+  }
+
+  "Rendering the VAT details page for a non mtd user" should {
+
+    lazy val view = views.html.vatDetails.details(nonMtdDetailsModel, Html("<nav>BTA Links</nav>"))
+    lazy implicit val document: Document = Jsoup.parse(view.body)
+
+    "have the mtd sign up section" which {
```

Comment:
The three tests in here are failing for me. It's most likely a feature switch issue. Try putting `mockConfig.features.mtdSignUp(true)` in the `beforeEach()` of `ViewBaseSpec`